cmd/utils.go

```diff
@@ -263,6 +264,62 @@ func getProjectDir(config *RootCommandConfig) (string, error) {
 	return config.ProjectDir, nil
 }
 
+func setProjectName(config *RootCommandConfig, projectName string) error {
+	dir, perr := getProjectDir(config)
+	if perr != nil {
+		return errors.Errorf("The current directory is not a valid appsody project. Run appsody init <stack> to create one: %v", perr)
+	}
+	appsodyConfig := filepath.Join(dir, ConfigFile)
+	v := viper.New()
+	v.SetConfigFile(appsodyConfig)
+	err := v.ReadInConfig()
```

Comment:
Do not need to read, since we are only writing

Reply:
We need ReadInConfig() for writing into the file as well.